crates/evm/src/interpreter.cairo

```
@@ -72,7 +72,8 @@ impl EVMInterpreterImpl of EVMInterpreterTrait {
 
         // Check if PC is not out of bounds.
         if pc >= bytecode_len {
-            return Result::Err(EVMError::InvalidProgramCounter(PC_OUT_OF_BOUNDS));
+            machine.set_stopped();
```

Already started to tackle this in #459 
But I think if pc == bytecode_len we stop,
if pc > bytecode_len, there's a problem, we should return an error,
though this should never happen, as jump forbids non-valid jumps

---

Let's get #459 merged before this PR then, as it's required to properly test